src/frontend/src/lib/components/icons/IconAstronautWithEyes.svelte

```diff
@@ -0,0 +1,27 @@
+<svg width="47" height="40" viewBox="0 0 47 40" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
`currentColor` for consistency?


but maybe at some point it will be some sort of dual color input (for example if we do dark theme), so no blocker

Reply:
let me try...

Reply:
It worked out but, not that particular property ;)

You can review the commit.

Reply:
https://github.com/dfinity/oisy-wallet/pull/2939/commits/d429a7a14098886fec60920ac5e1f27c8b78f040

Reply:
Right! Eheh!